packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -405,6 +428,13 @@ export class DataStores implements IDisposable {
                 }
             }));
 
+        summaryBuilder.getSummaryTree().stats = {
```

Comment:
I don't think this set works.. `SummaryTreeBuilder` was intentionally not meant to be edited like that.

If you follow the stack up, it's a shallow copy of the stats:
`SummaryTreeBuilder.getSummaryTree()` creates an object with `stats: this.stats`, and
`get SummaryTreeBuilder.stats()` returns `{ ...this.summaryStats }`, so it ends up being a deep copy.